airflow/www/views.py

```diff
@@ -2373,7 +2373,7 @@ def _clear_dag_tis(
 
     @expose("/clear", methods=["POST"])
     @auth.has_access_dag("PUT", DagAccessEntity.TASK_INSTANCE)
-    @action_logging
+    @action_logging(event="cleared")
```

Comment:
Should we have it as manually cleared?

Reply:
i am ok with either.  i think @bbovenzi does not like it.  i don't have a strong opinion.  but i think `cleared` is *definitely* better than `clear` :)